packages/dai/scripts/prepublish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ $(basename $(pwd)) != "dai" ]; then
+  echo "This script must be run from the dai directory."
+  exit
+fi
+
+yarn config set version-tag-prefix "dai-v"
+yarn config set version-git-message "dai-v%s"
+yarn version
+
+PACKAGE_VERSION=$(cat package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[",]//g')
+
+git push origin dev --no-verify
+git push origin "dai-v${PACKAGE_VERSION:1}"
+
+CWD=`dirname $0`
+rm -rf dist/*
```

Comment:
shouldn't we be using `build-backend.sh` here instead of copying it?

Reply:
Yeah, I was having issues with this script. I got it working now, though 👍